AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -50,6 +50,8 @@ disk.yandex.*##.mail360-banner
 !-----------------------------------------------
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/100335
+findjob.co.kr##.findall_banner
```

Comment:
It you have a single rule to the site in this file, you can add it without the task comment, but it's ok.